unfetter-discover/unfetter-ui
# Issue 955 stixpages

# problem
* need a way for admin types to create and manipulate assessment categories/groups

# solution
* add a categories stix page, allow list, create, edit, delete

# test
* visit the stix categories page
* view the list
* select create
* create new category
 * ensure you are routed back to the list page
 * ensure the success snackbar message
* delete a category from the list
  * ensure it is deleted
* select edit
  * ensure the page comes up with correct values
